azure-pipelines.yml

```diff
@@ -109,6 +109,7 @@ jobs:
       python -m pip install
       biopython
       chemfiles>=0.10
+      distopia>=0.2.0
```

Comment:
As above, until distopia becomes a core requirement it shouldn't be in our base Windows tests otherwise you'll end up not testing what's currently in core against windows